src/app/components-small/template-details/template-details.component.ts

```diff
@@ -93,4 +89,8 @@ export class TemplateDetailsComponent implements OnInit {
             }
         }
     }
+
+    getLocations(): { [group: string]: string } {
```

Comment:
just create a public object property for this and copy the object if you want to use it safely in the template:

```ts
public tabGroupNameOverrides = {...TAB_GROUP_NAME_OVERRIDES};
```